luigi_pipeline/seqr_sv_loading.py

```diff
@@ -37,6 +37,13 @@ class SeqrSVVariantMTTask(SeqrVCFToVariantMTTask):
     # NB: electing not to override import_vcf here eventhough the inherited args are slightly different
     # than from the old pipeline.
 
+    def import_dataset(self):
+        mt = self.import_vcf()
+        return mt.key_rows_by('rsid')
```

Comment:
If my understanding of all of this is correct, `rsid` will persist when writing the Variants and Genotypes mt even though it's not included in the annotations.

Reply:
So we want this keyed by `variantId`, which is derived from `rsid` but we don't want that duplicated. I think what we would want is `key_rows_by(variantId=mt.rsid)`

Reply:
Do we want to drop `variantId` when we export?  I think current behavior is to drop whatever we're using as keys when we export to elastic search.  

We also have [this annotation](https://github.com/broadinstitute/seqr-loading-pipelines/blob/master/luigi_pipeline/lib/model/sv_mt_schema.py#L211-L213) which we would need to remove if we're doing the annotation here.